templates/web/base/admin/update_edit.html

```diff
@@ -46,7 +46,7 @@
 [% END -%]
 </li>
 [% IF update.problem_state %]
-<li>[% tprintf(loc('Update changed problem state to %s'), update.problem_state) %]</li>
+<li>[% tprintf(loc('Update changed problem state to: %s'), update.problem_state) %]</li>
```

Comment:
I wouldn't change this text string as part of this, seems okay as is.
The line in the issue was referring to the fact that just because an update has `problem_state`, sadly that doesn't mean this update changed the state, because it could have just been a steady state entry from the last update.